python/src/wslink/server.py

```diff
@@ -192,48 +198,43 @@ def start_webserver(
     # Create default or custom ServerProtocol
     wslinkServer = protocol()
 
-    if disableLogging:
-        logging_level = None
-    elif options.debug:
-        logging_level = logging.DEBUG
-    else:
-        logging_level = logging.ERROR
-
-    if options.reverse_url:
-        server_config = {
-            "reverse_url": options.reverse_url,
-            "ws_protocol": wslinkServer,
-            "logging_level": logging_level,
-        }
-    else:
-        server_config = {
-            "host": options.host,
-            "port": options.port,
-            "timeout": options.timeout,
-            "logging_level": logging_level,
-        }
-
-        # Configure websocket endpoint
-        if not options.nows:
-            server_config["ws"] = {}
-            server_config["ws"][options.ws] = wslinkServer
-
-        # Configure default static route if --content requested
-        if len(options.content) > 0:
-            server_config["static"] = {}
-            # Static HTTP + WebSocket
-            server_config["static"]["/"] = options.content
+    server_config = {
```

Comment:
I think the merge from master did not do a good job here. Please revisit that change as we changed the structure here to support reverse connection and from the diff it is hard to tell what is happening now... My guess it should be a more compact change.